_data/icons/toki.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "ipfs://QmZzDBbiideNFaNFxfHxeuB6Y5uTC4LMpaoHBtpzBj9Qco",
+    "width": 512,
+    "height": 512,
```

Comment:
size does not seem to match:

```
identify QmZzDBbiideNFaNFxfHxeuB6Y5uTC4LMpaoHBtpzBj9Qco       07:54:22
QmZzDBbiideNFaNFxfHxeuB6Y5uTC4LMpaoHBtpzBj9Qco SVG 517x517 517x517+0+0 8-bit sRGB 12542B 0.000u 0:00.000

```

Reply:
That's odd. The svg is below and does not have any reference to 517. It uses a viewBox but does not explicitly set a width/height - making it dimensionless. What tool are you using to print the above? I can make sure that it matches.

Reply:
I've updated the commit. If this does not work, please let me know the tool generating the above so I can troubleshoot.